spec/pagseguro/transaction_request_spec.rb

```diff
@@ -120,8 +120,7 @@
       let(:raw_xml) { File.read("./spec/fixtures/invalid_code.xml") }
 
       it "does not create a transaction request" do
-        expect(transaction_request.create).to be_falsy
-        expect(transaction_request.code).to be_nil
+        expect { transaction_request.create }.not_to change { transaction_request.code }
```

Comment:
Yes, I understand that serialize the object and update your attributes. But in Rails, ActiveRecord check if object is stored (or persisted) in database based search by sql in database whether returns the value searched so the object exist.

Anyway I can't think about other way. I believe that we'll have not problem with this.